src/spikeinterface/widgets/utils.py

```diff
@@ -283,6 +284,10 @@ def make_units_table_from_sorting(sorting, units_table=None):
         values = sorting.get_property(col)
         if values.dtype.kind in "iuUSfb" and values.ndim == 1:
             units_table.loc[:, col] = values
+        else:
+            warn(
+                f"Property {col} not added to the units table because it has ndim > 1 or dtype not in 'iuUSfb'",
```

Comment:
```suggestion
                f"Property {col} not added to the units table because it has ndim > 1 or dtype not in 'iuUSfb'",
```

That error is fine for people that know numpy, but seems a bit too intense for new folks who aren't going to know what iuU etc means. Do we want succinct or would it be better to bring it down to a beginner level and list them out? I think this is an open question. I always prefer making it as simple as possible since the target audience is the neuroscientist and not the developer.

Reply:
Done :)

Reply:
Thanks :) since I don't use sorting view much is there someone better to doublecheck for correctness? It looks fine to me, but we can always be patient for a review by Sam.

Reply:
I'll ask him to check :)